### PR TITLE
Restructure activegate controller folder structure

### PR DESCRIPTION
**Pull request by StefanHauth**
# Description

Cleanup folder structure of activegate code.

## How can this be tested?
n/a

## Checklist
- [X] Unit tests have been updated/added
- [X] PR is labeled accordingly
- [X] I have read and understood the [contribution guidelines](/CONTRIBUTING.md)

### CODE REVIEW

**Comment by meik99** [comment]
I'll take a look at it

File: src/controllers/dynakube/activegate/capability/capability.go [hunk 1]
```diff
@@ -5,11 +5,11 @@ import (
 	"strings"
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/dynakube/activegate/statefulset"
 	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	MultiActiveGateName       = "activegate"
```
Comment (0sewa0) [inline src/controllers/dynakube/activegate/capability/capability.go:12]: was this removed because of circular dependency ?

**Review by gkrenn — CHANGES_REQUESTED** [review]
ActiveGate sts is not deployed if running in classicfullstack and activegate with 
```
    capabilities:
      - routing
      - kubernetes-monitoring
      - dynatrace-api
```
enabled

**Edit**: Can be closed, my previous tests influenced my testsetup. Tested again and everything works as expected 🥳

File: src/controllers/dynakube/activegate/capability/capability.go (hunk 1 above)
Comment (dt-team-kubernetes) [reply src/controllers/dynakube/activegate/capability/capability.go:12]: yes

**Review by gkrenn — DISMISSED** [review]

**Review by gkrenn — APPROVED** [review]

**Pull request merged**